src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -19,7 +19,34 @@ public class DocumentationBrowserViewModel : NotificationObject, IDisposable
         private const string BUILT_IN_CONTENT_FILE_NOT_FOUND_FILENAME = nameof(Resources.FileNotFound) + ".html";
         private const string BUILT_IN_CONTENT_NO_CONTENT_FILENAME = "NoContent.html";
         private const string SCRIPT_TAG_REGEX = @"<script[^>]*>[\s\S]*?</script>";
+        private const string DPISCRIPT = @"<script> function getDPIScale()
+        {
+            var dpi = 96.0;
+            if (window.screen.deviceXDPI != undefined)
+            {
+                dpi = window.screen.deviceXDPI;
+            }
+            else
+            {
+                var tmpNode = document.createElement('DIV');
+                tmpNode.style.cssText = 'width:1in;height:1in;position:absolute;left:0px;top:0px;z-index:99;visibility:hidden';
+                document.body.appendChild(tmpNode);
+                dpi = parseInt(tmpNode.offsetWidth);
```

Comment:
So what is this doing?

Reply:
it's changing the zoom style based on the dpi - if this was not done, then the text would be extremely small when using a high dpi screen with display scaling over 100%.

Reply:
This looks like it's just returning a `dpi` and the style is being set in `adaptDPI` based on the `dpi` value. What I was trying to understand is how this `tmpNode.offsetWidth`, whatever it is, is related to the `dpi`. Just curious.

Reply:
ah, sorry, this was implemented by lepus team - my understanding from them was that this was a standard polyfill to get the DPI of an HTML element. Using the value on the WPF side yielded incorrect results sometimes.

Reply:
It's actually a clever way of measuring the DPI. It's creating a `div` element with fixed size of 1 inch x 1 inch. The element is absolutely positioned so that it's not affected by anything else in the page, but it's visibility is hidden so we cannot see it either. The `offsetWidth` property gives the actual size in pixels of the element. Since the CSS size was specified as `1in` that's the amount of pixels that equate to 1 inch, which is the DPI.

Reply:
Clever indeed!